cdap-api/src/main/java/co/cask/cdap/api/customaction/CustomActionSpecification.java

```diff
@@ -44,4 +46,10 @@
    * @return an immutable set of {@link Dataset} name that are used by the {@link CustomAction}
    */
   Set<String> getDatasets();
+
+  /**
+   * @return RetryPolicy for remote calls or {@code null} if the CDAP default should be used.
+   */
+  @Nullable
```

Comment:
It's better not to return null. If the user didn't set it, this can return whatever the default is.